### PR TITLE
Largestnumber

### PR DESCRIPTION
**Pull request by mansi1014**
I have written a c program which finds largest among n natural numbers.

### CODE REVIEW

**Comment by Akkimaurya** [comment]
> I have written a c program which finds largest among n natural numbers.
@mansi1014 you wrote a program to check leap year.
not largest among n natural number

File: 5.c [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <conio.h>
+void main ()
+{
+    int y;
+   
+    printf ("Enter the year : ");
+    scanf("%d" , y);
```
Comment (Anjali-1801) [inline 5.c:8]: Please change y with &y

**Review by Anjali-1801 — COMMENTED** [review]
Segmentation fault.

File: 5.c (hunk 1 above)
Comment (sarvesh-0) [reply 5.c]: @Anjali-1801  and @mansi
Why is this PR not geeting merged? Its now more than a week. Any issue? 
Please let me know why its still pending.

Comment (Anjali-1801) [reply 5.c]: Sir there were few errors in the first program, those were resolved yesterday only. I was about to merge it today.

**Comment by mansi1014** [comment]
Good morning Sir,
Sir there were some indentation fault in the leap year program I have
resolved it yesterday. Anjali will probably merge it today.
Sorry for the delay sir.

**Pull request merged**